packages/app-utils/src/components/LoginNotifier.tsx

```diff
@@ -5,7 +5,7 @@ import { useEffect } from 'react';
 /**
  * Component that broadcasts a message when mounted. Should be mounted after the user has logged in.
  */
-export function LoginNotifier() {
+export function LoginNotifier(): null {
```

Comment:
Kind of weird just having a `null` return type here - I was going to suggest `ReactNode` as that includes `null`, but then that freaks out in `AuthBootstrap`. Looks like that issue is fixed in TS 5.1: https://devblogs.microsoft.com/typescript/announcing-typescript-5-1-beta/#decoupled-type-checking-between-jsx-elements-and-jsx-tag-types
If we were >=5.1, might consider using `ReactNode` instead.

Reply:
It looks like `JSX.Element | null` is currently the return type declared by `React.FC`. Maybe that new 5.1 type will address this, but seems like `<SomeComponent/>` does not work for things like `string` which is included in `ReactNode`. For now I think `JSX.Element | null` will be the most appropriate for component return types.